components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/protobuf/ProtoOpenAPI.java

```diff
@@ -104,7 +106,14 @@ void addOpenAPIPath(String path, String[] scopes, String throttlingTier) {
         if (openAPI.getPaths() == null) {
             openAPI.setPaths(new Paths());
         }
-        openAPI.getPaths().addPathItem(path, pathItem);
+        //as Responses object is mandatory
+        ApiResponse apiResponse = new ApiResponse();
+        apiResponse.setDescription("gRPC response");
```

Comment:
Lets use constants

Reply:
This is a description. So do we still need to have a constant?

Reply:
updated via https://github.com/wso2/product-microgateway/pull/1214/commits/abc899d72f6c15dc0996fa057fe99cc26ffb10af